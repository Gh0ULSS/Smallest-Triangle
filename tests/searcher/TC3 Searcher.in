46.52, -48.83
23.6, -34.2
37.95, 18.14
35.73, 49.98
-26.03, -16.19
20.82, -21.94
-23.67, -27.14
35.78, 37.45
29.76, -27.66