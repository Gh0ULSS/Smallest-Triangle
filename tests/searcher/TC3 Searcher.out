read 9 points
23.60, -34.20
29.76, -27.66
20.82, -21.94
This is a triangle